ci-scripts/dockerfiles/tutorials.dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Execute this from the top-level ReFrame source directory
+#
+
+
+FROM reframehpc/rfm-ci-base:lmod
+
+ENV _SPACK_VER=0.16
+ENV _EB_VER=4.4.1
+
+# ReFrame user
+RUN useradd -ms /bin/bash rfmuser
+
+USER rfmuser
+
+# Install Spack
+RUN git clone https://github.com/spack/spack ~/spack && \
```

Comment:
I get `==> Error: spack requires 'curl'. Make sure it is in your path.`
Do you get the same ?